test/e2e/framework/deploy/deploy.go

```diff
@@ -0,0 +1,115 @@
+// +build e2e
+
+package deploy
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/Azure/secrets-store-csi-driver-provider-azure/test/e2e/framework"
+	"github.com/Azure/secrets-store-csi-driver-provider-azure/test/e2e/framework/exec"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/yaml"
+
+	. "github.com/onsi/gomega"
+)
+
+var (
+	driverResourcePath        = "https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/deploy"
```

Comment:
With the helm charts we test a specific version of the driver as defined in the `requirements`. Should we do the same here too and use a tagged release url instead of the `master` one?

Reply:
makes sense. Updated to `v0.1.0`